{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -65,10 +65,10 @@ commands =
     {%- endif %}
 {%- else %}
     {%- if cookiecutter.test_matrix_separate_coverage|lower == 'yes' %}
-    nocov: {posargs:nosetests -v tests}
```

Comment:
This needs not be removed: it would collect the code in the `src` dir while normally it's imported from `site-packages`. That would lead to many subtle issues(like doubly executed module, reference issues, collection errors etc).